### PR TITLE
Feature/grafana node tolerations

### PR DESCRIPTION
**Pull request by yagonobre**

### CODE REVIEW

File: stable/grafana/templates/deployment.yaml [hunk 1]
```diff
@@ -23,6 +23,10 @@ spec:
     spec:
       nodeSelector:
 {{ toYaml .Values.server.nodeSelector | indent 12 }}
+  {{- if .Values.server.tolerations }}
+      tolerations:
+{{ toYaml .Values.server.tolerations | ident 12 }}
```
Comment (unguiculus) [inline stable/grafana/templates/deployment.yaml]: Hm, `indent 12` doesn't seem correct. The same goes for `nodeSelector`.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/grafana/templates/deployment.yaml (hunk 1 above)
Comment (yagonobre) [reply stable/grafana/templates/deployment.yaml]: you are right, changed to 8

**Comment by zanhsieh** [comment]
LGTM.

**Comment by unguiculus** [comment]
Please bump the chart version.

**Comment by yagonobre** [comment]
Done!

**Review by unguiculus — APPROVED** [review]

**Pull request merged**